hca/dss/upload_to_cloud.py

```diff
@@ -67,14 +67,22 @@ def upload_to_cloud(file_handles, staging_bucket, replica, from_cloud=False, log
                               In addition, even if this is set to True, a progress bar will not
                               be shown if (a) the logging level is not INFO or lower or (b) an
                               interactive session is not detected.
+    :param file_uuids: A list of UUIDs to use for the file handles uploaded.  Must be in the same order as
+                       the file handles.  Defaults to random UUIDs if not specified.  Ignored if "from_cloud=True".
```

Comment:
The ordering requirement is a bit of a pain. A mapping from uuid to handle would make more sense ...